.github/workflows/build.yml

```diff
@@ -5,38 +5,38 @@ on: push
 jobs:
   build:
     runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
     steps:
     - name: Checkout repository
       uses: actions/checkout@v2
 
-    - name: Get branch name
+    - name: Get version name
       shell: bash
-      run: echo "##[set-output name=name;]`echo $([ ${GITHUB_REF#refs/heads/} = "master" ] && echo 'latest' || echo ${GITHUB_REF#refs/heads/}) | sed 's/\//-/g'`"
+      run: echo "##[set-output name=name;]`echo ${GITHUB_REF#refs/tags/}`"
       id: branch
 
     - name: Build Docker Container
-      run: docker build -t ractf/core:${{ steps.branch.outputs.name }} -t ghcr.io/ractf/core:${{ steps.branch.outputs.name }} .
+      run: docker build -t ractf/core:${{ steps.branch.outputs.name }} -t ghcr.io/ractf/core:${{ steps.branch.outputs.name }} -t ractf/core:latest -t ghcr.io/ractf/core:latest .
 
     - name: Create Sentry release
       uses: getsentry/action-release@v1
-      if: github.ref == 'refs/heads/master'
       env:
         SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
         SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
         SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
 
     - name: Login to Docker Hub
-      if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop'
       run: docker login --username ractf --password ${{ secrets.password }}
 
     - name: Login to Github Container Registry
-      if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop'
       run: echo ${{ secrets.ghtoken }} | docker login ghcr.io -u ${{ secrets.ghuser }} --password-stdin
 
     - name: Push to Docker Hub
-      if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop'
-      run: docker push ractf/core:${{ steps.branch.outputs.name }}
+      run: |
+          docker push ractf/core:${{ steps.branch.outputs.name }}
+          docker push ractf/core:latest
```

Comment:
Does this syntax work? Not seen people do this before

Reply:
Do you mean for Docker, or for YAML?

Reply:
GitHub actions specifically, I wasn't aware you could just put multiple commands in by using a multi-line string

Reply:
See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun:
![image](https://user-images.githubusercontent.com/22679924/121956221-28810d00-cd61-11eb-8117-8600180018fc.png)

Reply:
Huh, you learn something everyday